src/message/outgoing-message.js

```diff
@@ -0,0 +1,67 @@
+const Transform = require('readable-stream').Transform;
+
+const { Packet, HEADER_LENGTH, isValidType } = require('../packet');
+
+const BufferList = require('bl');
+
+/**
+  OutgoingMessage
+
+  This class can be used to transform the raw contents of a single
+  TDS message into a stream of TDS packets.
+*/
+module.exports = class OutgoingMessage extends Transform {
+  constructor(type, resetConnection, packetSize, debug) {
+    super();
+
+    if (!isValidType(type)) {
+      throw new TypeError('"type" must be a a supported TDS message type');
+    }
+
+    if (typeof packetSize !== 'number' || packetSize <= HEADER_LENGTH) {
+      throw new TypeError(`"packetSize" must be a number greater than ${HEADER_LENGTH}`);
```

Comment:
This is on purpose, it's a ES6 template string. Notice the use of `${...}` inside the string.